testER/src/components/Popup.jsx

```diff
@@ -1,96 +1,39 @@
-import React from 'react'
```

Comment:
The diff is ugly here because of naming convention. The original content of this file got moved to `SubjectPopupContent.jsx`. 

This is now a lightweight wrapper that uses effect hooks to keep a mapbox popup's content in sync with the React app's state and output.